kmk/extensions/status_indicator.py

```diff
@@ -0,0 +1,21 @@
+from kmk.extensions.lock_status import LockStatus
+
+
+class Indicator(LockStatus):
+    def __init__(self):
+        super().__init__()
+        self.layers = []
+
+    def after_hid_send(self, sandbox):
+        backup = self.report
+        super().after_hid_send(sandbox)
+        if self.report != backup:
+            self.update()
```

Comment:
Not sure why this is necessary. You could just as easily check for a change in `status_indicator.get_caps_lock()` directly, or any other metric for that matter.